tests/test_decoder.py

```diff
@@ -734,3 +736,132 @@ def test_parse_error():
 
     with pytest.raises(pvl.decoder.ParseError):
         pvl.load(io.BytesIO(b'foo'))
+
```

Comment:
Exposed in 1f8f8aa and fixed in 700da87. It was a really simple and small bug. I just forgot to add the argument to the decoder when loading a filepath in [line 89](https://github.com/planetarypy/pvl/pull/29/commits/700da877b73d3819118fb6cbc0307b753226b074#diff-0a5de9c15bd399f5f184838723f4b310R89). Tests pass.